docs/content/components/collection/Listbox/listbox.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Listbox
+caption: A component for displaying a list of options and allows user to select one or more of them.
+badge: new
+---
+
+The `<Listbox>` helps you to build accessible listbox of components than can be styled as needed.
```

Comment:
I think you wanted to write:

The `<ListBox>` helps you build accessible listbox components that can be styled as needed.